articles/azure-developer-cli/azd-templates.md

```diff
@@ -88,7 +88,7 @@ Refer to the README in any of the following Azure Developer CLI enabled template
 
 ---
 
-For more community contributed templates, [browse GitHub using `azd-templates` as the topic]( https://github.com/topics/azd-templates).
+For more community contributed templates, check out our template gallery: [Awesome AZD](aka.ms/awesome-azd).
```

Comment:
```suggestion
For more community contributed templates, check out our template gallery: [Awesome AZD](https://aka.ms/awesome-azd).
```

This will still need the https:// to be a proper or fully formed URL for the build checks.